components/UserList.js

```diff
@@ -31,7 +25,13 @@ function UserList ({ summaryView, type }) {
     else router.push(`/manage?type=${type}&userId=${user.id}`)
   }
   const limit = 10
-  const url = `${_getUrl(type)}?offset=${offset}&limit=${limit}`
+  const url = `${getUserUrl(type)}?offset=${offset}&limit=${limit}`
```

Comment:
I would add some blank lines to visually separate the many declarations in this section.